### DISCUSSION TITLE
Question: Streaming over 80/443

### DISCUSSION THREAD

**Discussion by FreieProgrammierung** [AzuraCast/AzuraCast#6526] [Ideas]
I wonder if live streaming is possible over port 80 or 443? To have to open as few ports as possible in the firewall. I.e. that one is connected over a specific Mountpoint with the respective radio and streams over it.

Sorry for this English. I have used a translator.

**Comment by Moonbase59** [comment]
Sure, AzuraCast provides for that. My streams are all on 443 which has the advantage that you can embed a stream in a `https://` web page and won’t get mixed content warnings. Also good for keeping the # open ports down of course.

Only drawback: Most broadcasting clients _insist_ on streaming via `http://` usually to port(s) 8005 and/or 8006 (AzuraCast’s default for the 1st station), so you may have to open these if doing live streaming.

**Comment by BusterNeece** [comment]
If you're referring to streamers connecting *to* your installation, then currently that's only possible via the WebDJ feature, which uses Websockets instead of standard packets.

But yes, listeners can listen to any stream from the main 80/443 ports just by enabling the "Use Web Proxy" system setting.
